@xen-orchestra/lite/src/components/CollectionFilter.vue

```diff
@@ -9,9 +9,9 @@
       {{ filter }}
     </UiFilter>
 
-    <UiButton :icon="faPlus" class="add-filter" color="secondary" @click="open">
+    <UiActionButton :icon="faPlus" class="add-filter" @click="open" small>
```

Comment:
I do not understand the `small` prop. 
I am not seeing any `small: boolean` prop in the `UiActionButton` props definition.